Sources/TuistKit/Graph/GraphLoader.swift

```diff
@@ -45,9 +48,9 @@ class GraphLoader: GraphLoading {
     fileprivate func loadProject(path: AbsolutePath) throws -> Graph {
         let cache = GraphLoaderCache()
         let circularDetector = GraphCircularDetector()
-        let project = try Project.at(path, cache: cache, circularDetector: circularDetector)
+        let project = try Project.at(path, cache: cache, circularDetector: circularDetector, modelLoader: modelLoader)
```

Comment:
to minimize changes, keeping this as is for the moment, an alternate later on could be

`let project = modelLoader.loadProject(...)`